src/chapters/06-interactive-components.md

```diff
@@ -43,6 +54,8 @@ Ember will create an *[instance][TODO: link to instance]* of the class whenever
 
 Here, in the *[component's constructor][TODO: link to component's constructor]*, we *[initialized][TODO: link to initialized]* the *[instance variable][TODO: link to instance variable]* `this.isLarge` with the value `false`, since this is the default state that we want for our component.
 
+## Accessing Instance Variable States from Templates
```

Comment:
"Instance Variable States" is redundant, either "Instance Variables" or "Instance States" are fine (probably the latter).